windows/rngallery/rngallery.vcxproj

```diff
@@ -6,7 +6,7 @@
     <CppWinRTOptimized>true</CppWinRTOptimized>
     <CppWinRTRootNamespaceAutoMerge>true</CppWinRTRootNamespaceAutoMerge>
     <MinimalCoreWin>true</MinimalCoreWin>
-    <ProjectGuid>{f4d4ef0f-b724-41ed-b6ef-bcf8d91e4775}</ProjectGuid>
+    <ProjectGuid>{5e21afcf-aaf9-4c22-a858-a53bb8cde1fb}</ProjectGuid>
```

Comment:
Have we landed on steps to keep the project guid static when we run through a version update?

Reply:
For now there's no automatic way to keep the guid the same following a version update. If the developer wished to keep it the same they would have to manual undo the changes of the --overwrite script to restore the original guid information.

Reply:
It's here, and potentially could be backported to 0.67: https://github.com/microsoft/react-native-windows/pull/9453